hyperspy/io_plugins/msa.py

```diff
@@ -263,6 +262,13 @@ def parse_msa_string(string, filename=None):
     else:
         # Defaulting to EELS looks reasonable
         mapped.set_item('Signal.signal_type', 'EELS')
+    if 'YUNITS' in parameters.keys():
```

Comment:
Many times YUNITS are not defined but the SIGNALTYPE is. What about guessing the quantity from the signal type instead i.e. EELS -> Electrons (counts), EDS -> Xrays (counts)?